### ISSUE TITLE
Packages Ubuntu 24.04 - pkg_install4ubuntu_lkp.sh

### ISSUE THREAD

**Issue by aigualada** [PacktPublishing/Linux-Kernel-Programming_2E#10]
Greetings,
The script fails on Ubuntu 24.04 because the packages openjdk-22-jdk and python3-distutils are not available. I had to install the packages openjdk-21-jdk and python3-distutils-extra.

Thank you for the book.

**Comment by kaiwan** [comment]
Yes, things like this will keep happening as we use newer distros... keeping seperate scripts for each distro version is difficult (as would a switch-case approach within one script), so I just left it as a script addressed to the particular distro ver the book was written for.. Thanks for your comment!

**Comment by jmiguel-hdez** [comment]
@kaiwan  i think openjdk 22 is not present on 22.04.05  either. 
i don't know if it was a typo but it seems it was never part of ubuntu 22.04.
